test/validators/others.ts

```diff
@@ -0,0 +1,147 @@
+            [1, 2, 3, 4],
+            helpers.DataGenerator.shares(4),
+            minDepositAmount,
+            firstCluster.cluster
+        ));
+
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            helpers.DataGenerator.publicKey(2),
+            firstCluster.operatorIds,
+        )).to.emit(ssvNetworkContract, 'ValidatorExited')
+            .withArgs(helpers.DataGenerator.publicKey(2), firstCluster.operatorIds);
+    });
+
+    it('Exiting a removed validator reverts "ValidatorDoesNotExist"', async () => {
+        await ssvNetworkContract.connect(helpers.DB.owners[1]).removeValidator(
+            helpers.DataGenerator.publicKey(1),
+            firstCluster.operatorIds,
+            firstCluster.cluster
+        );
+
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            helpers.DataGenerator.publicKey(1),
+            firstCluster.operatorIds
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'ValidatorDoesNotExist');
+    });
+
+    it('Exiting a non-existing validator reverts "ValidatorDoesNotExist"', async () => {
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            helpers.DataGenerator.publicKey(12),
+            firstCluster.operatorIds
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'ValidatorDoesNotExist');
+    });
+
+    it('Exiting a validator with empty operator list reverts "IncorrectValidatorState"', async () => {
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            helpers.DataGenerator.publicKey(1),
+            []
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'IncorrectValidatorState');
+    });
+
+    it('Exiting a validator with empty public key reverts "ValidatorDoesNotExist"', async () => {
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            '0x',
+            firstCluster.operatorIds
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'ValidatorDoesNotExist');
+    });
+
+    it('Exiting a validator using the wrong account reverts "ValidatorDoesNotExist"', async () => {
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).exitValidator(
+            helpers.DataGenerator.publicKey(1),
+            firstCluster.operatorIds
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'ValidatorDoesNotExist');
+    });
+
+    it('Exiting a validator with the wrong operator setup reverts "IncorrectValidatorState"', async () => {
+        await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).exitValidator(
+            helpers.DataGenerator.publicKey(1),
+            [4, 3, 2, 1]
+        )).to.be.revertedWithCustomError(ssvNetworkContract, 'IncorrectValidatorState');
+    });
+
+    it('Exiting a validator with the wrong operator setup reverts "IncorrectValidatorState"', async () => {
```

Comment:
Please add another test where you use operators 1,2,3,5 with title of:
Exiting a validator with incorrect operators (incorrect operator ids with valid amount) reverts with "IncorrectValidatorState"